headers/types/ground_mode/ground_mode.h

```diff
@@ -774,4 +774,198 @@ struct swap_shop_inventory_data {
 };
 ASSERT_SIZE(struct swap_shop_inventory_data, 18132);
 
+struct ground_bg_sub_struct_4 {
+    int16_t field_0x0;
+    int16_t field_0x2;
+    const void* unk4;
```

Comment:
How do we know this is const? Also, unknown pointers should be `undefined*` and use the Ghidra naming convention `field_0x4`

Reply:
Now that you mention it, `const` isn't really needed in `pmdsky-debug`, so I'll just remove it from my changes.